docs/docs/cli-governance.md

```diff
@@ -37,7 +37,7 @@ The values within this code can be changed in order to alter the minimum deposit
 ```
 # Create initial proposal by uploading the JSON file
 # this is signed by a key 'proposer' that provides a portion of the current threshold deposit
-fetchd tx gov submit-proposal --proposal ~/json_path/proposal.json --deposit <deposit_value> --from proposer
+fetchd tx gov submit-proposal --proposal ~/json_path/proposal.json --from proposer
```

Comment:
:+1:

I think we also need to update the above json snippet to reflect this, it must be from some old version where the deposit field wasn't included in it. Here's an up to date one (extracted from `fetchd tx gov submit-proposal param-change --help`)
```
{
  "title": "Staking Param Change",
  "description": "Update max validators",
  "changes": [
    {
      "subspace": "staking",
      "key": "MaxValidators",
      "value": 105
    }
  ],
  "deposit": "1000000000000000000atestfet"
}
```

Reply:
Done